test/lib/commands/audit.js

```diff
@@ -1486,6 +1539,50 @@ t.test('audit signatures', async t => {
     t.matchSnapshot(joinedOutput())
   })
 
+  t.test('third-party registry with sub-path', async t => {
```

Comment:
Very nice 👍  would we need to handle a trailing slash on the path? e.g. `https://verdaccio-clone.org/npm/`

Reply:
Good call. Will add a test case to make sure that's handled properly.